nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -70,7 +70,19 @@ in {
       '';
     };
 
-    nginx.enable = mkEnableOption "nginx vhost management";
+    nginx = {
+      enable = mkEnableOption "nginx vhost management";
+      addr = mkOption {
```

Comment:
Should we really duplicate this or rather add documentation how to access/extend the vhost documentation of nginx?

```
nginx.enable = mkOption {
  type = types.bool;
  default = false;
  description = ''
    Enable nginx vhost management.
    Further nginx configuration can be done by adapting <literal>nginx.virtualHosts.&lt;hostName &gt;</literal>.
    See <insert-link-to-virtualHosts-doc> for further information.
  '';
}:
```

Otherwise we also end up duplicating options like letsencrypt.

Reply:
:+1: for documenting on how to extend this from the nginx module, instead of duplicating options.